[FEATURE] Add additional Layout Templates for 'News' Extension

dominiksenti opened the pull request:
The changes add some additional News Template Layouts
-List view: Media Objects, Thumbnails, Timeline
-Detail view: Bootstrap with Standard Social Sharing or integration of
rx_shariff (Extension has to be installed manually)

Last week I created a pull request with this features, but there are
some errors in the files (No newline at end of file). Now I hope it
works...

![list view media objects](https://cloud.githubusercontent.com/assets/15846537/11501794/28d9171c-9837-11e5-994e-c1bea5cff188.png)
![list view thumbnails](https://cloud.githubusercontent.com/assets/15846537/11501796/2da55396-9837-11e5-8987-f6ab8e499553.png)
![list view timeline on desktops](https://cloud.githubusercontent.com/assets/15846537/11501800/31159720-9837-11e5-94a3-def4032e8673.png)
![list view timeline on mobiles](https://cloud.githubusercontent.com/assets/15846537/11501801/34eb24d2-9837-11e5-9133-935c048e8f26.png)

dominiksenti committed:
[FEATURE] Add additional Layout Templates for 'News' Extension

The changes add some additional News Template Layouts
-List view: Media Objects, Thumbnails, Timeline
-Detail view: Bootstrap with Standard Social Sharing or integration of
rx_shariff (Extension has to be installed manually)

Last week I created a pull request with this features, but there are
some errors in the files (No newline at end of file). Now I hope it
works...

jozefspisiak commented:
@dmh Can we merge this?

dmh commented:
No, we can't, because it was not adapt for t3kit.
**But soon we will add similar functionality for news.**
@dominiksenti sorry that it was not explained to you in time.

dominiksenti commented:
@dmh thx for your answer. I don't understand, what you mean exactly with "it was not adapt for t3kit". Is it a styling question or something else? No problem with not merging my pull request, but as i am quite new on programming extensions, i would like to understand what i could do better the next time... Thx for your reply, if you wan't to.

dmh commented:
@dominiksenti 
This PR is pretty old, and I don't remember what was a reason, that's why you got such a limited answer. In general it was bad communication, I hope it will not happen again. 

I will try to check it again and explain reasons to you.

Thanks!

dominiksenti commented:
@dmh Thx for the reply. Yes, that's true: the PR is pretty old. First time when i see the t3kit project i was really excited. Many years i tried several concepts of templating included the themes project. But all of this solutions aren't the right things for me... Short word: t3kit was exactly what i searched. Congrats! Since then, i introduced a new website with t3kit. For the missing things like special news layouts, contact cards and other special things, i made my own extension which is based on t3kit (like your bluemountain project).

Because i am from the german part of Switzerland i am very interested in a german translation. The decision to make PRs is based on the Open Source thought. I wanted to give you something back. If you are interested, i can help you with this. Maybe you have to say me which files you would like to have translated and which "rules" i have to use.

As i said in the beginning of my comment: a lot of time is gone since my PR. Maybe you are interested to see the result of my project, which i realized with t3kit?

If so you can have a look here:
http://www.jugendkatechese.ch

The implementation of News:
http://www.jugendkatechese.ch/blog 

A sample of my contact cards:
http://www.jugendkatechese.ch/themen/h1-identitat/

In addition i have implemented a variant of a Simple HTML DOM Parser for Medienliste (media list). You will find this on the same page as the contact card (the fourth tab in the main content area). This renders the content of this original source with the whole layout:
http://www.relimedia.ch/themensammlung/ich-identitaet/

Last what i would like draw attention to, is the website navigation.

So, i would be happy if i can contribute something, so that t3kit become even better.

Thanks for your attention and i would be delighted to hear from you.

Dominik
